tests/std/tests/P0896R4_counted_iterator/test.cpp

```diff
@@ -251,6 +268,10 @@ struct instantiator {
                 assert(iter1 <=> iter1 == strong_ordering::equal);
                 assert(iter1 <=> iter1 == strong_ordering::equivalent);
             }
+            { // spaceship value initialized

```

Comment:
```suggestion
            { // spaceship value-initialized
```

Reply:
@CaseyCarter I think you didn't apply this one, but it's probably not worth resetting testing.

Reply:
Ack, missed one. I'll apply this after the test run completes so we know they're clean. I